.github/workflows/benchmark_signed_integer.yml

```diff
@@ -53,6 +62,27 @@ jobs:
         run: |
           echo "op_flavor=${{ toJSON(env.OP_FLAVOR) }}" >> "${GITHUB_OUTPUT}"
 
+      - name: Default benchmark type
+        if: github.event_name != 'workflow_dispatch'
+        run: |
+          echo "BENCH_TYPE=[\"latency\"]" >> "${GITHUB_ENV}"
+
+      - name: Set one benchmark type
+        if: inputs.bench_type != 'both'
+        run: |
+          echo "BENCH_TYPE=[\"${{ inputs.bench_type }}\"]" >> "${GITHUB_ENV}"
+
+      - name: Both benchmarks types
+        if: inputs.bench_type == 'both'
+        run: |
+          echo "BENCH_TYPE=[\"latency\", \"throughput\"]" >> "${GITHUB_ENV}"
```

Comment:
can be a single small shell script

if [[ "${{ inputs.bench_type }}" == "both" ]]; then
    todo
else
    todo
fi